dokka-runners/dokka-gradle-plugin/src/main/kotlin/workers/WorkerIsolationProperties.kt

```diff
@@ -38,7 +38,7 @@ sealed class WorkerIsolation {
      * Create a Worker using [process isolation][WorkerExecutor.processIsolation].
      *
      * Gradle will launch
-     * [new Worker Daemon](https://docs.gradle.org/8.5/userguide/worker_api.html#creating_a_worker_daemon)
+     * [new Worker Daemon](https://docs.gradle.org/8.10/userguide/worker_api.html#worker-daemons),
```

Comment:
```suggestion
     * [new Worker Daemon](https://docs.gradle.org/current/userguide/worker_api.html#worker-daemons),
```

Reply:
I think it's better to use a fixed link to avoid linkrot (as demonstrated by the link changing between 8.5 and 8.10.